src/frontend/screens/ToolManager/index.tsx

```diff
@@ -0,0 +1,171 @@
+import './index.scss'
+
+import ContextProvider from 'frontend/state/ContextProvider'
+import { UpdateComponent } from 'frontend/components/UI'
+
+import React, { lazy, useContext, useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Tab, Tabs } from '@mui/material'
+import {
+  StoreIpc,
+  toolDownloaderInfoStore
+} from 'frontend/helpers/electronStores'
+import { ToolVersionInfo, Type } from 'common/types/toolmanager'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faSyncAlt } from '@fortawesome/free-solid-svg-icons'
+
+const ToolItem = lazy(
```

Comment:
lazy loading of react components triggers the `Suspense` component, generating a weird flicker of the screen showing a `loading` screen for a moment

we shouldn't use lazy load of components unless there's a really good performance benefit

Reply:
this was marked as resolved, it's like this intentionally? we've had issues with lazy loaded components

Reply:
nevermind, it was showing me an outdated/cached diff, this was changed